cypress/plugins/index.js

```diff
@@ -11,10 +11,14 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 const path = require('path')
+const cypressTypeScriptPreprocessor = require('./cy-ts-preprocessor')
 
 module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+
+  on('file:preprocessor', cypressTypeScriptPreprocessor)
+
```

Comment:
Where is it that TS preprocessor errors will show up, where will I see them if there are any?